cpp/api/include/trtorch/trtorch.h

```diff
@@ -90,7 +90,7 @@ struct TRTORCH_API ExtraInfo {
      * This class is compatable with c10::DataTypes (but will check for TRT support)
      * so there should not be a reason that you need to use this type explictly.
      */
-    class DataType {
+    class TRTORCH_API DataType {
```

Comment:
Is this needed for windows platform?

Reply:
I see a class DeviceType as well without TRTORCH_API

Reply:
Yes, on Windows it complained about DataType not being exported.

Reply:
ok